catalogs/sources/gtfs/schedule/pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.json

```diff
@@ -18,5 +19,11 @@
         "direct_download": "https://mkuran.pl/gtfs/tristar.zip",
         "latest": "https://storage.googleapis.com/storage/v1/b/mdb-latest/o/pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.zip?alt=media",
         "license": "http://www.opendefinition.org/licenses/cc-by"
-    }
+    },
```

Comment:
@A-dead-pixel Upon further investigation, it looks like there is a replacement feed for Gdynia: http://api.zdiz.gdynia.pl/pt/gtfs.zip

Could you add it and then also include it in the redirect list?

Reply:
Ah, you're right! This [site](https://otwartedane.gdynia.pl/pl/dataset/informacje-o-rozkladach-jazdy-i-lokalizacji-przystankow) with more info about that feed misled me, as it says the last update was in 2023, though the file contains recent data.
I added it.